src/transformation/builtins/math.ts

```diff
@@ -33,12 +34,12 @@ export function transformMathCall(context: TransformationContext, node: Property
 
     const expressionName = expression.name.text;
     switch (expressionName) {
-        // math.tan(x / y)
+        // math.atan(y, x) Lua 5.3 only
```

Comment:
"Lua 5.3 **only**" sounds like it would throw when it's not 5.3
```suggestion
        // math.atan2(y, x)
        // Lua 5.3: math.atan(y, x)
```

Reply:
```suggestion
        // Lua 5.3: math.atan(y, x)
        // Otherwise: math.atan2(y, x)
```